system/scripts/zone/core/calc_skill.cs

```diff
@@ -323,6 +324,11 @@ public float SCR_GET_SklSpdRate(Skill skill)
 			byBuff = spdBm / 1000f;
 		}
 
+		if (skill.IsMonsterSkill)
```

Comment:
Hm, are the checks for monster skills for this purpose potentially flawed? I don't know if there are monsters who use player skills or if there are monster skills that can somehow be used by players (outside of custom behavior), but the way the options are phrased, it sounds like they apply to all enemies/monsters, regardless of what kinds of skills they use. Would it potentially make more sense to check if the _owner_ is a monster, instead of relying on the skill type?

Reply:
I think generally the purpose of this adjustment is to deal with the fact that the skill factors and animations speeds for player skills vs monster skills are egregiously unbalanced.  If a monster has access to player skills, those skills are likely to be dozens of times more powerful than its standard attack (while also having vastly better hitboxes and attack speeds), so further boosts to them are likely not required.

Reply:
Perhaps that's the case, but I think it's bad form to add options that work in some very particular way and disregard other potential use cases that seem like they should work, but don't because you thought they're unlikely to apply. Know what I mean? I might change this, because if there's no reason not to do it, it's better to cover your bases.